utils/appliance/__init__.py

```diff
@@ -1662,6 +1662,7 @@ def log_raise(exception_class, message):
             else:
                 log_callback(
                     'Appliance must be restarted before the netapp functionality can be used.')
+        del self.is_storage_enabled
```

Comment:
is it certain we always have this as an local attribute, else we should use the helper function to clear cached properties